lib/src/services/ds_bottom_sheet.service.dart

```diff
@@ -28,22 +29,16 @@ class DSBottomSheetService {
                   children: [
                     StickyHeader(
                       overlapHeaders: false,
-                      header: _grabber(),
-                      content: Padding(
-                        padding: const EdgeInsets.symmetric(horizontal: 16.0),
-                        child: _buildChild(padding),
-                      ),
+                      header: _grabber(), Text('teste'),
```

Comment:
check this widget 'text'.. it's not compiling

Reply:
Adjusted.